admin/bump_version.py

```diff
@@ -152,6 +153,37 @@ def updateCallOfReleaseChecklist(filename: str, lines: List[str]):
     return newLines
 
 
+def updateChangelog(filename: str, lines: List[str]):
+    newLines = []
+
+    nextLine = None
+    for line in lines:
+        if line == "Next Release":
+            line = f"{VERSION} ({time.strftime('%d %B %Y')})"
+            nextLine = "-" * len(line)
+        elif nextLine:
+            line = nextLine
+            nextLine = None
+        newLines.append(line)
```

Comment:
This part feels somehow complicated. If we can trust that the `----` line always exist (and the current code does trust this because it always overwrites the next line), then it could be simplified to:

```suggestion
    for line in lines:
        if line == "Next Release":
            newLines.append(f"{VERSION} ({time.strftime('%d %B %Y')})")
        else:
            newLines.append(line)
```

Reply:
The problem ist that the length of the string must be the same. If not, you'll get a RST error.

Reply:
Ok, that makes total sense (your code, not the RST syntax…).